terraform/hydra-nixpkgs.tf

```diff
@@ -2,7 +2,7 @@ locals {
   jobset = {
     cuda = {
       name                 = "cuda"
-      description          = "Testing CUDA support. Come help the CUDA team! https://nixos.org/community/teams/cuda/"
```

Comment:
https://github.com/nix-community/infra/pull/1575

I'll add a page to our docs for cuda and have a link to the cuda there.